api/v1beta1/hyperconverged_types.go

```diff
@@ -439,6 +439,12 @@ type HyperConvergedFeatureGates struct {
 	// +default=false
 	DeployKubeSecondaryDNS *bool `json:"deployKubeSecondaryDNS,omitempty"`
 
+	// Deploy KubevirtIpamController by CNAO
```

Comment:
Please add some more details here. What's KubevirtIpamController? even add a reference.

Reply:
we have it here
docs/cluster-configuration.md
and i just followed the other structs, do you still think it is required?

Reply:
Still, we need it in the PR description. If we missed it in previous PR, that too bad, but we do need it

Reply:
We don't need it to be too long, but this is an input to the CRD, and part of the API. it should be very clear to the user. the docs/cluster-configuration.md is for human readable text.